apps/site/src/components/Footer/Footer.tsx

```diff
@@ -47,20 +52,65 @@ const SOCIALS: Social[] = [
 export default function Footer() {
 	return (
 		<footer className={styles.footer}>
-			<p className={styles.text}>Made with &lt;3 in Irvine, CA - Hack @ UCI</p>
-			<div className="text-center">
-				{SOCIALS.map(({ icon, link, alt }) => (
+			<div className={clsx(styles.left_footer, styles.hack_icon)}>
+				{LEFT_SOCIALS.map(({ icon, link, alt }) => (
 					<a key={link} href={link} target="_blank">
 						<Image
 							src={icon.src}
 							alt={alt}
-							width={45}
-							height={45}
+							width={55}
+							height={55}
 							className={styles.socials}
 						/>
 					</a>
 				))}
 			</div>
+			<div className={styles.buttons}>
+				<div id={styles.box}>
+					<div className={clsx(styles.left_footer, styles.links)}>
+						<Link
+							href="/#"
+							className={styles.left_footer_text}
+						>
+							Home
+						</Link>
+						{/* <Link
+							href="https://zothacks-2023.devpost.com/"
+							className={styles.left_footer_text}
+							target="_blank"
+						>
+							Devpost
+						</Link> */}
+						<Link
+							href="https://zothacks-2023.devpost.com/"
+							className={styles.left_footer_text}
+							target="_blank"
+						>
+							About Us
+						</Link>
+					</div>
+					<div className={styles.right_footer}>
+						{RIGHT_SOCIALS.map(({ icon, link, alt }) => (
+							<a key={link} href={link} target="_blank">
+								<Image
+									src={icon.src}
+									alt={alt}
+									width={40}
+									height={40}
+									className={styles.socials}
+								/>
+							</a>
+						))}
+					</div>
+				</div>
+
+				<a href="https://docs.google.com/forms/d/e/1FAIpQLSeZN1njLZC9N_JE9eMYbtSZ95O1SafaZVLddTNReln6eZR7Vg/viewform" 
```

Comment:
Chore: Rather than putting the form URL directly here, we can use our vanity URL `/apply` and let the site redirects handle it.